### PR TITLE
enable black and minors chores

### PR DESCRIPTION
**Pull request by thoroc**
This is to enable black when the PR runs. So committers don't have to.

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -18,4 +18,4 @@ def lint(session):
     session.run("flake8", *args)
 
 
-nox.options.sessions = ["lint"]
+nox.options.sessions = ["black", "lint"]
```
Comment (ixi2101) [inline noxfile.py:21]: Whilst this does run black, the job won't be able to push the changes it makes as main is protected

**Review by ixi2101 — CHANGES_REQUESTED** [review]

**Review by Risewill23 — APPROVED** [review]

**Comment by ixi2101** [comment]
I think a better solution would be
https://pre-commit.com/ hook to run black

File: noxfile.py (hunk 1 above)
Comment (thoroc) [reply noxfile.py:21]: well that's problematic. It does address the issue locally.

**Comment by thoroc** [comment]
pre-commit could be considered, but it might be a bit of an overkill with the deadline looming.

**Pull request closed without merging**